docs/CHANGELOG.asciidoc

```diff
@@ -29,6 +30,65 @@ Review important information about the {kib} 8.x releases.
 
 --
 
+[[release-notes-8.3.1]]
+== {kib} 8.3.1
+
+coming::[8.3.1]
+
+Review the following information about the {kib} 8.3.1 release.
+
+[float]
+[[breaking-changes-8.3.1]]
+=== Breaking change
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade, review the breaking change, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+There are no breaking changes in {kib} 8.3.1.
+
+// end::notable-breaking-changes[]
+
+To review the breaking changes in previous versions, refer to the following:
+
+{kibana-ref-all}/8.2/release-notes-8.2.0.html#breaking-changes-8.2.0[8.2.0] | {kibana-ref-all}/8.1/release-notes-8.1.0.html#breaking-changes-8.1.0[8.1.0] | {kibana-ref-all}/8.0/release-notes-8.0.0.html#breaking-changes-8.0.0[8.0.0] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc2.html#breaking-changes-8.0.0-rc2[8.0.0-rc2] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc1.html#breaking-changes-8.0.0-rc1[8.0.0-rc1] | {kibana-ref-all}/8.0/release-notes-8.0.0-beta1.html#breaking-changes-8.0.0-beta1[8.0.0-beta1] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha2.html#breaking-changes-8.0.0-alpha2[8.0.0-alpha2] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha1.html#breaking-changes-8.0.0-alpha1[8.0.0-alpha1]
+
+[float]
+[[enhancement-v8.3.1]]
+=== Enhancements
+Operations::
+Adds EsArchiver datastream support {kibana-pull}132853[#132853]
+
+[float]
+[[fixes-v8.3.1]]
+=== Bug fixes
+Alerting::
+Prevent negative snooze intervals {kibana-pull}134935[#134935]
+
+Elastic Security::
+For the Elastic Security 8.3.1 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+* Fixes dropping select all {kibana-pull}135124[#135124]
+* Improves bulk actions for more than 10k agents {kibana-pull}134565[#134565]
+
+Infrastructure::
+Query persistent queue size for metricbeat documents {kibana-pull}134569[#134569]
+
+Machine Learning::
+* Fixes put job endpoint when payload contains datafeed {kibana-pull}134986[#134986]
+* Fixes trained model map associating wrong model to job {kibana-pull}134849[#134849]
+* Use time range when validating datafeed preview {kibana-pull}134073[#134073]
+
+Observability::
+* Fixes a bug that displayed a toast error when deleting a rule {kibana-pull}135132[#135132]
+* Fixes viewInAppUrl for custom metrics for Inventory Threshold Rule {kibana-pull}134114[#134114]
+
+Platform::
+* Fixes an issue where copying the same saved object to a different space multiple times could cause duplicates to be created, even when using the "check for existing objects" option {kibana-pull}135358[#135358]
```

Comment:
Sorry for the confusion @KOTungseth, we had a conversation above, can you apply this other suggested change instead?

```suggestion
* Fixes an issue where importing/copying the same saved object to the same space multiple times using the "Check for existing objects" option could fail or cause duplicates to be created {kibana-pull}135358[#135358]
```